Content.Server/Disease/Components/DiseaseCarrierComponent.cs

```diff
@@ -32,5 +32,12 @@ public sealed class DiseaseCarrierComponent : Component
         /// <summary>
         [ViewVariables(VVAccess.ReadWrite)]
         public List<DiseasePrototype> AllDiseases => PastDiseases.Concat(Diseases).ToList();
+
+        /// <summary>
+        /// Whether or not the entity experiences any
+        /// symptoms from their diseases.
+        /// </summary>
+        [DataField("noSymptoms")]
+        public bool NoSymptoms = false;
```

Comment:
Shouldn't this be a per-disease thing, or an event? Would anything else even use it besides rat king?

Reply:
This could generally be used for any entity that needs to spread a disease while not being subject to it's effects. I don't see how you would make it an event, but yeah it could be modified to be per disease. For most cases, it probably would just be for gimmicky enemies that need to spread diseases, so there isn't a need for too much depth. I can still modify it though.